backend/src/main/java/com/google/sps/notifs/EmailNotifier.java

```diff
@@ -161,16 +158,21 @@ public void notifyUser() throws MessagingException, GeneralSecurityException, IO
                 + getRecipientName()
                 + " Please Join your Ad-Lib meeting via the link below : \n"
                 + " http://meet.google.com/new");
-    NetHttpTransport HTTP_TRANSPORT = GoogleNetHttpTransport.newTrustedTransport();
-    Gmail service =
-        new Gmail.Builder(HTTP_TRANSPORT, JSON_FACTORY, getCredentials(HTTP_TRANSPORT))
-            .setApplicationName(APPLICATION_NAME)
-            .build();
     Message messageWithEmail = createMessageWithEmail(email);
-    service.users().messages().send("me", messageWithEmail).execute();
+    getService().users().messages().send("me", messageWithEmail).execute();
```

Comment:
Any benefit to using a getter instead of directly referencing `service` here?

Reply:
In the function we were setting service as the gmail Builder,because of your feedback I thought that having a getter would be the way to access all of our gmail api calls